apiserver/facades/client/highavailability/highavailability.go

```diff
@@ -156,6 +162,33 @@ func (api *HighAvailabilityAPI) enableHASingle(st *state.State, spec params.Cont
 	return controllersChanges(changes), nil
 }
 
+// getReferenceController looks up the ideal controller to use as a reference for Constraints and Series
+func getReferenceController(st *state.State, machineIds []string) (*state.Machine, error) {
+	// Sort the controller IDs from low to high and take the first.
```

Comment:
I assume this is just code being moved, right?

Reply:
Yes.